### ISSUE TITLE
Unable to install with provided "BuildCommand" and unable to run UpdateRemotePlugins

### ISSUE THREAD

**Issue by FvD** [euclio/vim-markdown-composer#2]
When I try to build with the provided `BuildComposer` function either outside or inside the call plug# begin and call plug#end section in nvimrc I will get the following error:

    Error detected while processing /home/myuser/.nvimrc:
    line   76:
    E15: Invalid expression: )
    E116: Invalid arguments for function <SNR>8_add
    line   77:
    E20: Mark not set
    line   78:
    E492: Not an editor command:     }
    Press ENTER or type command to continue

However, if I only include:

     Plug 'euclio/vim-markdown-composer'

The installation will proceed and I can run `cargo build --release` inside the folder.

However, after this manual installation the `:UpdateRemotePlugins` step fails with the following message:

```
Error detected while processing function <SNR>50_UpdateRemotePlugins..<SNR>50_RegistrationCommands..remot
e#host#Require..<SNR>50_RequirePythonHost:                                                               
line   21:                                                                                               
E605: Exception not caught: Failed to load Python host. You can try to see what happened by starting Neov
im with the environment variable $NVIM_PYTHON_LOG_FILE set to a file and opening the generated log file. 
Also, the host stderr will be available in Neovim log, so it may contain useful information. See also ~/.
nvimlog.                                                                                                 
Error detected while processing function <SNR>50_UpdateRemotePlugins..<SNR>50_RegistrationCommands..remot
e#host#Require:                                                                                          
line   10:                                                                                               
E171: Missing :endif                                                                                     
Error detected while processing function <SNR>50_UpdateRemotePlugins:                                    
line    5:                                                                                               
E171: Missing :endif 
```

**Comment by euclio** [comment]
Oops, there's a typo in the build instructions. Fixed.

As for the remote plugins issue, that sounds like a problem with your remote plugin host setup. What OS are you running? And can you give me the output of `pip --version`?

**Comment by FvD** [comment]
Thanks @euclio. After reading your reply I thought that perhaps It was because I had not updated the python neovim package in a while. But that does not seem to be the cause of the issue.

My pip version is:

     pip 7.1.0 from /usr/local/lib/python2.7/dist-packages (python 2.7)

I keep neovim up to date on Ubuntu using the neovim ppa (ppa:neovim-ppa/unstable).

**Comment by FvD** [comment]
It was caused by a unmet python3 dependency. I had only ever installed neovim with pip, so there was no neovim package available for neovim in python three.

The steps to fix this on Ubuntu 14.04 LTS:

1. Install pip3: 
    `sudo apt-get install python3-pip`

2. Update pip3
    `pip3 install pip --upgrade`

3. Install neovim for python3
    `pip3 install neovim`

And now there are no more error messages when the `:UpdateRemotePlugins` is called.
